src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -79,10 +79,6 @@ private static Type[] InitializeSkipInterfaceAttributeList()
         {
             return new Type[]
             {
-#if FEATURE_SKIP_INTERFACE
```

Comment:
This is not what I had intended when I wrote the issue.

> For the first two, we should remove the #if and enable the code it is protecting. These are left-over from pre-standard2.0 days when some APIs weren't available. As part of enabling this code we should add unit tests for the behavior.

To be explicit, I meant:

```diff
- #if FEATURE_SKIP_INTERFACE
                typeof(System.Runtime.InteropServices.GuidAttribute),
                typeof(System.Runtime.InteropServices.InterfaceTypeAttribute)
- #endif
```

Which means this code would now look like:

```C#
        private static Type[] InitializeSkipInterfaceAttributeList()
        {
            return new Type[]
            {
                typeof(System.Runtime.InteropServices.GuidAttribute),
                typeof(System.Runtime.InteropServices.InterfaceTypeAttribute)
                typeof(System.Runtime.InteropServices.ComVisibleAttribute),
            };
        }
```

See https://github.com/dotnet/corefx/pull/12977 for how we removed these `#if`s previously.

The added unit tests should tests should fail when using the current product code, and pass with the new changes. They should be testing that `GuidAttribute` and `InterfaceTypeAttribute` are skipped correctly.